contributing_to_docs/term_glossary.adoc

```diff
@@ -391,75 +423,72 @@ menu that does not have hover text or a given name or label in the web console.
 == P
```

Comment:
Should we add a note for persistent volume and persistent volume claim similar to custom resource definition (CRD)?

> Usage: persistent volume and persistent volume claims
> 
> These are commonly abbreviated as a PV and a PVC.
> 
> Developers can use a PVC to request a PV resource without having specific knowledge of the underlying storage infrastructure.

Reply:
Maybe a generic entry for "volume" too?

Reply:
Thanks @bobfuru I added entries for PV and PVC.

@adellape if you have a good description to use for just "volume", I'd be happy to add. Otherwise we can always add in a separate PR.